### DISCUSSION TITLE
How can I archive an entire subreddit minus media?

### DISCUSSION THREAD

**Discussion by ocean-haiyang** [ArchiveBox/ArchiveBox#1614] [General]
Looking to archive /selfhosted

**Comment by TomK32** [comment]
While the depth flag works as expected and filled the queue, it also added other subreddits. Btw, old.reddit.com still works and has rss feeds. Sadly when I tried the "Generic RSS" parser it didn't find anything.

There's some info for you here #1022 

Two small features that I see useful for this case (without RSS)

- mirror archives see #191
- allow setting parameters for pagination

**Reply by TomLucidor** [reply]
Can someone throw in Pullpush/Pushshift for older content? Probably adjacent to the fresh-downloading
